praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/rest/MeldezeitraumController.java

```diff
@@ -53,4 +55,11 @@ public List<MeldezeitraumDto> getMeldezeitraeume(@RequestParam(name = "period",
         }
         return meldezeitraumService.getAllMeldezeitraeume();
     }
+
+    @DeleteMapping
+    @Transactional
+    @PreAuthorize("hasRole('ROLE_' + T(de.muenchen.oss.praktikumsplaner.security.AuthoritiesEnum).AUSBILDUNGSLEITUNG.name())")
+    public void deleteMeldezeitraum(@RequestParam(name = "id") UUID id) {
```

Comment:
After looking at our current API i think this should be a PathVariable. Because currently ids that directly relate to the endpoint are provided as PathVariable and other info such as the NwkId in the Assignment endpoint are provided as RequestParam